samples/excel/89-preview-apis/events-worksheet-activated.yaml

```diff
@@ -0,0 +1,156 @@
+
+                await context.sync();
+
+                if (sheet.items.length > 1) {
+                    let lastSheet = sheet.items[sheet.items.length - 1];
+                    lastSheet.delete();
+
+                    console.log(`Deleted worksheet named "${lastSheet.name}"`);
+                }
+                else {
+                    console.log("Unable to delete worksheet.");
+                }
+            });
+        }
+
+        async function setup() {
+            await Excel.run(async (context) => {
+                let sheet = await OfficeHelpers.ExcelUtilities
+                    .forceCreateSheet(context.workbook, "Sample");
+                let salesTable = sheet.tables.add('A1:E1', true);
+                salesTable.name = "SalesTable";
+
+                salesTable.getHeaderRowRange().values = [["Product", "Qtr1", "Qtr2", "Qtr3", "Qtr4"]];
+
+                salesTable.rows.add(null, [
+                    ["Frames", 5000, 7000, 6544, 4377],
+                    ["Saddles", 400, 323, 276, 651],
+                    ["Brake levers", 12000, 8766, 8456, 9812],
+                    ["Chains", 1550, 1088, 692, 853],
+                    ["Mirrors", 225, 600, 923, 544],
+                    ["Spokes", 6005, 7634, 4589, 8765]
+                ]);
+
+                if (Office.context.requirements.isSetSupported("ExcelApi", 1.7)) {
+                    sheet.getUsedRange().format.autofitColumns();
+                    sheet.getUsedRange().format.autofitRows();
+                }
+
+                sheet.activate();
+
+                await context.sync();
+            });
+        }
+
+        /** Default helper for invoking an action and handling errors. */
+        async function tryCatch(callback) {
+            try {
+                await callback();
+            }
+            catch (error) {
+                OfficeHelpers.UI.notify(error);
+                OfficeHelpers.Utilities.log(error);
+            }
+        }
+
+
+
+    language: typescript
+template:
+    content: |
+        <section class="ms-font-m">
+            <p>This sample shows how to add the worksheet activated and deactivated event handlers.</p>
+        </section>
+
+        <section class="setup ms-font-m">
+            <h3>Set up</h3>
+            <button id="setup" class="ms-Button">
+                <span class="ms-Button-label">Add sample data</span>
+            </button>
+        </section>
+
+        <section class="samples ms-font-m">
+            <h3>Try it out</h3>
+            <button id="activate-event-handler" class="ms-Button">
+                <span class="ms-Button-label">Register on activated event handler</span>
+            </button>
+        </section>
+
+        <section class="samples ms-font-m">
+            <button id="deactivate-event-handler" class="ms-Button">
+                <span class="ms-Button-label">Register on deactivated event handler</span>
+            </button>
+        </section>
+
```

Comment:
Have text here that explains that deleting the current worksheet triggers the deactivate event for it and the activate event for the preceding worksheet.